packages/docs/docs/dev/reference.mdx

```diff
@@ -194,55 +155,20 @@ The events enum `FuelWalletEvents` can be imported from the `@fuel-wallet/sdk` p
 The `fuel` object has an `events` property which is an enum of all the events that can be listened to. <br />
 The `on` method takes two arguments, the event name and a callback function. The callback function receives data associated with the event.
 
-### Connection
-
-This event is triggered when the connection status change between the Wallet and the application. The callback function receives the `connection` object.
-
-<CodeImport
-  file="../../src/hooks/useIsConnected.tsx"
-  commentBlock="eventAccountChanges"
-/>
-
-### Accounts
-
-This event is triggered when the list of connected accounts to the application change. The callback function receives the `account` string.
-
-<CodeImport
-  file="../../examples/events/Accounts.tsx"
-  commentBlock="eventAccountChanges"
-/>
-
-### Current Account
-
-This event is triggered when the current account is changed in the Fuel Wallet Extension. The callback function receives the `account` address.
-
-This event is only triggered if the application has access to the current account address. Otherwise, it is not triggered.
-
-<CodeImport
-  file="../../examples/events/CurrentAccount.tsx"
-  commentBlock="eventCurrentAccount"
-/>
-
-### Network
-
-This event is triggered when the client connects to a different network. The callback function receives the `network` object.
-
-<CodeImport file="../../examples/events/Network.tsx" commentBlock="network" />
-
-### CurrentConnector
-
-This event is triggered when the Current Connector is updated. The callback function receives the `FuelWalletConnector` object.
-
-<CodeImport
-  file="../../examples/Connectors.tsx"
-  commentBlock="eventCurrentConnector"
-/>
-
-### Connectors
-
-This event is triggered when the Connectors list is updated. The callback function receives the list of connectors, `Array<FuelWalletConnector>`.
-
-<CodeImport
-  file="../../examples/Connectors.tsx"
-  commentBlock="eventConnectors"
-/>
+```ts
+fuel.on(fuel.events.connection, (connectionState) => {
+  console.log(connectionState);
+});
+```
+
+### Events
+
+| Event              | Trigger                   | Params                 |
+| ------------------ | ------------------------- | ---------------------- |
+| `connectors`       | connectors change         | `Array<FuelConnector>` |
+| `currentConnector` | currentConnector change   | `FuelConnector`        |
+| `accounts`         | accounts available change | `Array<string>`        |
+| `currentAccount`   | current account change    | `string \| null`       |
+| `connection`       | connection state change   | `boolean`              |
+| `currentNetwork`   | cuurent network change    | `Network`              |
+| `assets`           | assets metadata change    | `Array<Asset>`         |
```

Comment:
Chart looks very odd when generated
<img width="1147" alt="Screenshot 2023-11-22 at 5 02 57 PM" src="https://github.com/FuelLabs/fuels-wallet/assets/106365423/31c70fae-a017-475a-b8bd-fcd4b26b7c04">
Should match the docs-hub charts
<img width="755" alt="Screenshot 2023-11-22 at 5 06 28 PM" src="https://github.com/FuelLabs/fuels-wallet/assets/106365423/6ae714d5-e7de-471e-a7ac-6e06cef542fd">